pylib/tests/test_collection.py

```diff
@@ -143,10 +143,6 @@ def test_furigana():
     n["Front"] = "foo[sound:abc.mp3]"
     n.flush()
     assert "anki:play" in c.q(reload=True)
-    # it shouldn't throw an error while people are editing
-    m["tmpls"][0]["qfmt"] = "{{kana:}}"
-    mm.save(m)
-    c.q(reload=True)
```

Comment:
I've already asked if you are okay with enforcing valid field names, but just to be sure: It's okay to remove this test and reject something like `{{kana:}}`?

Reply:
the empty field name is a special case used by add-ons; we need to keep supporting that
https://github.com/ankitects/anki-addons/blob/5f775143c2cf906f689bac6d83f47493494e5e0d/demos/field_filter/__init__.py#L22